src/components/common/Dialogs/CreatePaymentDialog/CreatePaymentDialog.tsx

```diff
@@ -45,18 +43,21 @@ const CreatePaymentDialog = ({
   const colonyWatchers =
     colony?.watchers?.items.filter(notNull).map((item) => item.user) || [];
   const { isVotingReputationEnabled } = enabledExtensionData;
+  const { networkInverseFee } = useNetworkInverseFee();
 
   const actionType =
     !isForce && isVotingReputationEnabled
       ? ActionTypes.MOTION_EXPENDITURE_PAYMENT
       : ActionTypes.ACTION_EXPENDITURE_PAYMENT;
 
+  const validationSchema = getValidationSchema(colony);
+
+  type FormValues = InferType<typeof validationSchema>;
```

Comment:
As per Will's comment in #320, this can be moved outside the component and read:
```js
type FormValues = InferType<ReturnType<typeof getValidationSchema>>;
```